guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc

```diff
@@ -279,6 +279,6 @@ endif::[]
 
 * *Scripting with Hammer and API* – {Project} provides a command line tool called Hammer that provides a CLI equivalent to the majority of web UI procedures.
 In addition, you can use the access to the {Project} API to write automation scripts in a selected programming language.
-ifndef::orcharhino[]
-For more information see the {MultiBaseURL}hammer_cli_guide/[_Hammer CLI Guide_] and {MultiBaseURL}api_guide/[_API Guide_].
+ifdef::satellite[]
+For more information see the {HammerGuideURL}[_{HammerGuideTitle}_] and {APIGuideURL}[_{APIGuideTitle}_].
```

Comment:
```suggestion
For more information, see {HammerGuideURL}[{HammerGuideTitle}] and {APIGuideURL}[{APIGuideTitle}].
```

Reply:
I also noticed this part wasn't spotted in the emphasis unification.